assets/js/params/common/AllNoneButtons.js

```diff
@@ -1,32 +1,41 @@
 import React from 'react'
 import PropTypes from 'prop-types'
+import { Trans, t } from '@lingui/macro'
+import { withI18n } from '@lingui/react'
 
-export default function AllNoneButtons ({ isReadOnly, onClickAll, onClickNone }) {
+export function AllNoneButtons ({ isReadOnly, onClickAll, onClickNone, i18n }) {
   return (
     <div className='all-none-buttons'>
       <button
         disabled={isReadOnly}
         type='button'
         name='refine-select-all'
-        title='Select All'
+        title={i18n._(t('js.params.common.AllNoneButtons.selectAll.hoverText')`Select All`)}
         onClick={onClickAll}
       >
-        All
+        <Trans id='js.params.common.AllNoneButtons.selectAll.button'>All</Trans>
       </button>
       <button
         disabled={isReadOnly}
         type='button'
         name='refine-select-none'
-        title='Select None'
+        title={i18n._(t('js.params.common.AllNoneButtons.selectNone.hoverText')`Select None`)}
         onClick={onClickNone}
       >
-        None
+        <Trans id='js.params.common.AllNoneButtons.selectNone.button'>None</Trans>
       </button>
     </div>
   )
 }
-AllNoneButtons.propTypes = {
-  isReadOnly: PropTypes.bool.isRequired,
-  onClickNone: PropTypes.func.isRequired, // func() => undefined
-  onClickAll: PropTypes.func.isRequired // func() => undefined
-}
+AllNoneButtons.propTypes =
```

Comment:
Please move the `{` to this line (and undo your changes to indentation of the following lines).